### PR TITLE
Initial Schema

### PR DESCRIPTION
**Pull request by chrisdesoto**

### CODE REVIEW

File: server/schema/schema.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
+"users":
```
Comment (JeyzerMC) [inline server/schema/schema.json]: I asked in the wrong PR, but what is the benefits of having both each schema and a container schema?

File: server/schema/volunteers.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+	"full_name": "{{chance.first()}} {{chance.last()}}",
+	"phone": "{{chance.phone()}}",
+	"email": "{{chance.email()}}",
+	"shirt_size": "{{util.sample(['X-Small', 'Small', 'Medium', 'Large', 'X-Large'])}}",
+	"availability": ""
```
Comment (JeyzerMC) [inline server/schema/volunteers.json]: Should that be an array since the event will be over the span of 3 days?

Comment (adriandarian) [reply server/schema/volunteers.json]: look at the typeform I made for the volunteer listing, I added the time slots we are looking for there and a list of days
we should have those as two separate fields

File: server/schema/schema.json (hunk 1 above)
Comment (adriandarian) [reply server/schema/schema.json]: what do you mean @JeyzerMC

Comment (JeyzerMC) [reply server/schema/schema.json]: Like what is this specific file used for? It's a concatenation of all the other schemas, that's why

Comment (chrisdesoto) [reply server/schema/schema.json]: It was easier to share with the rest of the team. My shell script requires them to be different files to mock the data though so I added both.

**Review by adriandarian — APPROVED** [review]

**Pull request merged**